src/main/java/org/apache/commons/release/plugin/mojos/CommonsDistributionStagingMojo.java

```diff
@@ -380,7 +380,7 @@ private File copyReleaseNotesToWorkingDirectory() throws MojoExecutionException
      * <code>${basedir}/target/commons-release-plugin/scm/signature-validator.sh</code>.
      *
      * @return the {@link File} for the signature-validator.sh
-     * @throws MojoExecutionException
+     * @throws MojoExecutionException if an error occurs during the resource is copy
```

Comment:
I think it could be "is copied", or "is being copied"… though not a native speaker, so safer is someone else takes a look perhaps.

Reply:
Hi @kinow 
Thank you for the tip

changed for --> `@throws MojoExecutionException if an error occurs during the resource is being copied`

Reply:
Sorry, my mistake @arturobernalg . It now should be "if an error occurs when the resource is being copied". I think "when" or "while" fit in here.

Reply:
I didn't pay attention to the rest of the text when I suggested the modification. I can edit & merge it if you'd like me to

Reply:
Hi @kinow 
Don't worry.  Change.
Ty